### ISSUE TITLE
Windows: loading savegames, starting new mod games broken

### ISSUE THREAD

**Issue by DominusExult** [exult/exult#229]
One of the July 20th commits by @marzojr  broke both.
When you open the savegame gump and select an existing savegame it will say that the savegame can’t be read. Saving a new one will result in the same „broken“ state.

Starting a new game of a mod will crash Exult.

I have limited access to a Windows machine right now, so the github snapshots were the only way to pinpoint the problem

**Comment by eliyo** [comment]
I made a few debugging and found out the "unreadable savegame" error is due to Flex file checking (Magic number not found) in Game_window::get_saveinfo() calling Flex::is_flex(). Inspecting the savegame procedure, Game_window::save_gamedat() is calling save_gamedat_zip() if HAVE_ZIP_SUPPORT is defined (which is the case in my build). However save_gamedat_zip() doesn't seem to implement the Flex file writing. If commenting out the lines inside #ifdef HAVE_ZIP_SUPPORT into Game_window::save_gamedat(), then saving will be made using Flex, and this fixes the "unreadable savegame" issue.
I found another issue while testing is when pressing ESC key on the intro sequence of BG (in town, like while Iolo speaks), then Exult crashes.
